### DISCUSSION TITLE
Attack Styles - Context Menu Toggle

### DISCUSSION THREAD

**Discussion by lastgenangel** [runelite/runelite#16902] [Ideas]
New to this so apologies for poor formatting

When the "Attack Styles" plugin is enabled, I'd love a fast enable/disable toggle under the combat options context menu. 
![image](https://github.com/runelite/runelite/assets/141840737/c947b60c-f2ac-4107-8872-d15192411ba5)

The design wouldn't stray from options that already exist both officially and 3rd party.
![image](https://github.com/runelite/runelite/assets/141840737/0918904a-4f35-4184-9c7a-8d130041d99c)
![image](https://github.com/runelite/runelite/assets/141840737/f7077d6c-9ce4-474d-ad86-78ba5e41718a)

While this might not seem entirely useful, it would help a lot during certain activities or while playing multiple accounts, as some accounts benefit from the plugin, and others don't need it.  When opening multiple clients, the enabled plugins are uniform between each client, which leads to having to enable or disable the plugin across multiple clients. That can be worked around with the new plugin profiles feature, but this option would also give ease of use.

I understand this isn't high priority, but it would improve the (or at least my personal) experience when playing on multiple clients.

I'd appreciate any thought or insight on the matter.
Thanks.

**Comment by Nightfirecat** [comment]
I recommend using config profiles for this. In the future, it will likely be easier to associate config profiles with ingame accounts (see https://github.com/runelite/runelite/pull/16749 as a possible implementation), so that would work better than needing to constantly toggle these settings.
